opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactoryTest.java

```diff
@@ -369,322 +389,274 @@ public void constructDatetime_fromCustomFormat() {
   }
 
   @Test
   public void constructArrayOfDoubles() {
-    assertEquals(new ExprCollectionValue(
-            List.of(doubleValue(9.1928374756D), doubleValue(4.987654321D))),
+    assertEquals(
+        new ExprCollectionValue(List.of(doubleValue(9.1928374756D), doubleValue(4.987654321D))),
         constructFromObject("arrayV", List.of(9.1928374756D, 4.987654321D)));
   }
 
   @Test
   public void constructArrayOfBooleans() {
-    assertEquals(new ExprCollectionValue(
-            List.of(booleanValue(true), booleanValue(false))),
+    assertEquals(
+        new ExprCollectionValue(List.of(booleanValue(true), booleanValue(false))),
         constructFromObject("arrayV", List.of(true, false)));
   }
 
   @Test
   public void constructNestedObjectArrayNode() {
-    assertEquals(collectionValue(
-        List.of(
-            Map.of("count", 1),
-            Map.of("count", 2)
-        )),
-        tupleValueWithArraySupport("{\"nestedV\":[{\"count\":1},{\"count\":2}]}")
-            .get("nestedV"));
+    assertEquals(
+        collectionValue(List.of(Map.of("count", 1), Map.of("count", 2))),
+        tupleValueWithArraySupport("{\"nestedV\":[{\"count\":1},{\"count\":2}]}").get("nestedV"));
   }
 
   @Test
   public void constructNestedObjectArrayOfObjectArraysNode() {
     assertEquals(
         collectionValue(
             List.of(
-                Map.of("year",
+                Map.of(
+                    "year",
                     List.of(
                         Map.of("timeV", new ExprTimeValue("09:07:42")),
-                        Map.of("timeV", new ExprTimeValue("09:07:42"))
-                    )
-                ),
-                Map.of("year",
+                        Map.of("timeV", new ExprTimeValue("09:07:42")))),
+                Map.of(
+                    "year",
                     List.of(
                         Map.of("timeV", new ExprTimeValue("09:07:42")),
-                        Map.of("timeV", new ExprTimeValue("09:07:42"))
-                    )
-                )
-            )
-        ),
+                        Map.of("timeV", new ExprTimeValue("09:07:42")))))),
         tupleValueWithArraySupport(
-            "{\"deepNestedV\":"
-                        + "["
-                          + "{\"year\":"
-                            + "["
-                              + "{\"timeV\":\"09:07:42\"},"
-                              + "{\"timeV\":\"09:07:42\"}"
-                            + "]"
-                          + "},"
-                          + "{\"year\":"
-                            + "["
-                              + "{\"timeV\":\"09:07:42\"},"
-                              + "{\"timeV\":\"09:07:42\"}"
-                            + "]"
-                          + "}"
-                        + "]"
-                      + "}")
+                "{\"deepNestedV\":"
+                    + "["
+                    + "{\"year\":"
```

Comment:
Can you add spacing to avoid json being flat?

Reply:
Won't that cause the tests to fail?

Reply:
Shouldn't be. Something like
```
                "{\"deepNestedV\":"
                    + "  ["
                    + "    {\"year\":"
                    + "      ["
                    + "        {\"timeV\":\"09:07:42\"},"
                    + "        {\"timeV\":\"09:07:42\"}"
....
```

Reply:
Do the tests compare the exact string or just the content?

Reply:
@Yury-Fridlyand Resolved.

Reply:
`OpenSearchExprValueFactory` uses a json parser, so if a whitespace causes something to crash - that is a real bug.
thanks for fixing that!